tests/test_carrier_account.py

```diff
@@ -0,0 +1,75 @@
+import pytest
+
+import easypost
+
+
+def generate_carrier_account():
```

Comment:
We had this discussion yesterday. Since this helper function is only being used in `test_carrier_account.py`, we should be putting it in `confest.py`

Reply:
I misunderstood what you were asking. Because this is `static data`, it belongs in `conftest.py` along with the other fixtures of static data. If this function actually did something (aside from returning a dictionary) and was only used here; then yes, having this helper function in this file would make sense but that's not the case here.